src/connector.ts

```diff
@@ -66,7 +66,7 @@ class CloudSQLInstanceMap extends Map {
       return;
     }
     const connectionInstance = await CloudSQLInstance.getCloudSQLInstance({
-      connectionType: type,
+      connectionType: ipType,
```

Comment:
@shubha-rajan sorry for the extra work but could you also replace `connectionType` with `ipType` in `./src/cloud-sql-instance.ts` so that this line can be:

```suggestion
      ipType,
```

Reply:
tbh using `connectionType` was a mistake from my end in the original implementation and I wanted to fix that 😅

Reply:
fixed!

Reply:
thank you! 🙌